spec/oas3.fetched/openapi/public/paths/virtual-circuit/id/events.yaml

```diff
@@ -1,43 +0,0 @@
-get:
```

Comment:
This file was incorrect + redundant to spec/oas3.fetched/openapi/public/paths/virtual-circuits/id/events.yaml, which remains.